README.md

```diff
@@ -8,11 +8,11 @@ Always strive for specificity over brevity. The more specific our class selector
 
 ### Syntax
 
-We use the SCSS flavor of SASS to preprocess our CSS.
+We use the SCSS flavor of SASS to preprocess our CSS. You might want to use [scss-lint](https://github.com/brigade/scss-lint) with the configuration provided in this repository.
```

Comment:
@spicymagpie Depending on platform, https://github.com/sasstools/sass-lint is usually a better choice since it supports Sass & SCSS syntax *and* leverages LibSass which is signifcantly much faster accordingly. 

Accordingly, it's often seen as a better choice, but there's nothing wrong with naming both—with it likely being the first one listed instead of this one or a Ruby Sass - LibSass distinction.